tests/main/views/test_drafts.py

```diff
@@ -994,14 +994,14 @@ def test_drafts_made_from_services_are_deleted_when_published(self):
         assert self.service_count() == 1
         assert self.draft_service_count() == 0
 
-    @mock.patch('app.models.main.generate_new_service_id')
-    def test_service_id_collisions_should_be_handled(self, generate_new_service_id):
+    @mock.patch('app.models.main.random_positive_external_id')
+    def test_service_id_collisions_should_be_handled(self, random_positive_external_id):
         # Return the same ID a few times (cause collisions) and then return a different one.
-        generate_new_service_id.side_effect = (
-            '1234567890123457',
-            '1234567890123457',
-            '1234567890123457',
-            '2222222222222222',
+        random_positive_external_id.side_effect = (
+            '123456789012345',
+            '123456789012345',
+            '123456789012345',
+            '222222222222222',
```

Comment:
I've changed these IDs only so that they match the length of randomly generated IDs.

Reply:
Yeah this must have been from the good-old-days of 16 digit IDs that everyone hated.